### ISSUE TITLE
Regular users cannot concatenate files

### ISSUE THREAD

**Issue by apetkau** [phac-nml/irida#1203]
## Describe the bug
Regular users (not admins) cannot concatenate sequence files within a sample. Administrators can concatenate files.

## Steps to reproduce the problem
What were you doing when you encountered the problem?
1. Log in as a regular user (`User` role but a `Manager` on a project).
2. Go into a sample and select files to concatenate.
3. You will get an error message. With the following lines in the log file:

```
2022-02-21 14:03:43.300 ERROR 30679 --- [nio-8080-exec-9] c.c.b.i.r.web.samples.SamplesController  : Error concatenating files:

ca.corefacility.bioinformatics.irida.exceptions.ConcatenateException: Could not open input file for reading
        at ca.corefacility.bioinformatics.irida.processing.concatenate.SequencingObjectConcatenator.appendToFile(SequencingObjectConcatenator.java:52) ~[classes/:22
.01.1]
        at ca.corefacility.bioinformatics.irida.processing.concatenate.impl.SequenceFilePairConcatenator.concatenateFiles(SequenceFilePairConcatenator.java:59) ~[cl
asses/:22.01.1]
        at ca.corefacility.bioinformatics.irida.processing.concatenate.impl.SequenceFilePairConcatenator.concatenateFiles(SequenceFilePairConcatenator.java:17) ~[cl
asses/:22.01.1]
        at ca.corefacility.bioinformatics.irida.service.impl.SequencingObjectServiceImpl.concatenateSequences(SequencingObjectServiceImpl.java:248) ~[classes/:22.01
.1]

[...]

Caused by: java.nio.file.NoSuchFileException: 13/1/08-5578-small_S1_L001_R1_001.fastq.gz
```

## Expected behaviour
I expected to be able to concatenate files.

## Additional context

Screenshot of the error message:

![image](https://user-images.githubusercontent.com/200517/155019909-2266d41e-471c-42f7-ace0-366295a8bdc0.png)

Rest of the exception in the logs:
[irida-logs.txt](https://github.com/phac-nml/irida/files/8111841/irida-logs.txt)

**Comment by apetkau** [comment]
Related to #1200

**Comment by ericenns** [comment]
Fixed in #1202
